apps/golden-sample-app/src/app/ach-positive-pay/ach-positive-pay-journey-bundle.module.ts

```diff
@@ -1,5 +1,5 @@
 import { NgModule } from '@angular/core';
-import { AchPositivePayJourneyModule } from 'ach-positive-pay-journey';
+import { AchPositivePayJourneyModule } from '@libs/ach-positive-pay';
```

Comment:
Is this a local import?

Reply:
This is how imports are used from libs to apps in Nx.